tests/test_measurements.py

```diff
@@ -665,6 +665,37 @@ def test_sample_values(self, qubit_device, tol):
         # they square to 1
         assert np.allclose(s1**2, 1, atol=tol, rtol=0)
 
+    @pytest.mark.parametrize("seed", range(0, 10))
+    @pytest.mark.parametrize("nwires", range(1, 11))
+    def test_sample_variations(self, qubit_device, nwires, seed):
+        """Tests if `sample(wires)` returns correct statistics."""
+        shots = 100000
```

Comment:
Isn't it too many shots? :)

Reply:
Errors vary a lot w.r.t. `nwires`, `seed` so I had to use a fairly large number of shots, but this is not too expensive I reckon.

Reply:
The `tol` set below looks quite strict. I think that's one of the reason it requires larger shot number.

Reply:
Not a blocker and will let you decide.

Reply:
I'll play with it a bit more.